docs/process/labels.md

```diff
@@ -57,6 +57,23 @@ implementing features:
 | [:mag_right:][user-research]      | `user-research`      | Issues that may benefit from user interviews, validations, and/or usability testing |
 | [:mag_right:][needs-design-input] | `needs-design-input` | Issues that require design input from the core team before the work can be started |
 
+### Epics
+
+As we work on new parts of the application, or refining existing workflows, we use
+these labels to track issues that arise as part of the development process.
+
+This label is intended to make searching for issues related to an epic easier, and a
+project board should be used to organize and prioritize the work within an epic
+based on the upcoming release schedule.
+
+Each label should be removed from the repository when the project board is closed, so
+these labels do not become overused and add noise to the issue tracker.
+
+|                              | Label name     |  Description |
+| ---------------------------- | -------------- |  ----------- |
+| [:mag_right:][epic:rebase]   | `epic:rebase`  | Supporting rebase flows in the app |
+| [:mag_right:][epic:stashing] | `epic:stashing`| Supporting stashing uncommitted changes in the app |
```

Comment:
i'd like to try out sprinkling some emojis in our labels. how about something like this for stashing? i think it would really help with visual scanning.

```suggestion
| [:mag_right:][epic:stashing] | `📄 epic:stashing`| Supporting stashing uncommitted changes in the app |
```

Reply:
Personally, I prefer colour to visually differentiate these from other labels. I can't imagine us having more than 2-3 of these on the go, but I'm open to other suggestions.

I'm curious about how this affects the URLs generated (they contain the label name) - how unfriendly does this change make them?

Reply:
makes sense, we could do both color and emoji. i might try out the url thing, but regardless, let's not consider this suggestion blocking in any sense. :)